rastervision/data/scene_config.py

```diff
@@ -36,6 +36,12 @@ def create_scene(self, task_config: TaskConfig, tmp_dir: str) -> Scene:
         raster_source = self.raster_source.create_source(tmp_dir)
 
         extent = raster_source.get_extent()
+        if self.label_store and hasattr(
+                self.label_store,
+                'vector_output') and self.label_store.vector_output:
+            task_config.affine_transform = raster_source.get_affine_transform()
```

Comment:
It seems like the `task_config` is an odd place to stick the `affine_transform` since it's something that's associated with the raster source in the scene and not the task (which is constant across a set of tasks).  Instead, maybe you can just pass the `affine_transform` to the label_store directly.  It also might make sense to just make `affine_transform` a part of the crs_transformer, but I'm not sure about that. It would eliminate the issue of having to pass an extra thing along, and it fits with the role of the crs_transformer, but I haven't considered all the consequences of that change.